www/src/pages/index.js

```diff
@@ -1,246 +1,5 @@
-              <Anchor>Overlay</Anchor>
-            </h2>
-            <p
-              dangerouslySetInnerHTML={{
-                __html: OverlayMetadata.description.childMarkdownRemark.html,
-              }}
-            />
-            <Playground codeText={OverlaySource} />
-            <PropTable component="Overlay" metadata={OverlayMetadata} />
-          </section>
-
-          <section>
-            <h2 className="page-header">
-              <Anchor>Dropdown</Anchor>
-            </h2>
-            <p
-              dangerouslySetInnerHTML={{
-                __html: DropdownMetadata.description.childMarkdownRemark.html,
-              }}
-            />
-            <Playground codeText={DropdownSource} />
-            <PropTable component="Dropdown" metadata={DropdownMetadata} />
-            <PropTable
-              title="Dropdown.Menu"
-              component="ReactOverlaysDropdownMenu"
-              metadata={DropdownMenuMetadata}
-            />
-            <PropTable
-              title="Dropdown.Toggle"
-              component="ReactOverlaysDropdownToggle"
-              metadata={DropdownToggleMetadata}
-            />
-          </section>
-          <section>
-            <h2 className="page-header">
-              <Anchor id="root-close-wrapper">useRootClose</Anchor>
-            </h2>
-            <p
-              dangerouslySetInnerHTML={{
-                __html: useRootCloseDocs.description.childMarkdownRemark.html,
-              }}
-            />
-            <Playground codeText={useRootCloseSource} />
-            <HookDocumentation docs={useRootCloseDocs} />
-          </section>
-          <section>
-            <h2 className="page-header">
-              <Anchor id="use-popper">usePopper</Anchor>
-            </h2>
-            <p
-              dangerouslySetInnerHTML={{
-                __html: usePopperDocs.description.childMarkdownRemark.html,
-              }}
-            />
-
-            <HookDocumentation docs={usePopperDocs} />
-          </section>
-          <section>
-            <h2 className="page-header">
-              <Anchor>Transitions</Anchor>
-            </h2>
-            <p>
-              Animation of components is handled by <code>transition</code>{' '}
-              props. If a component accepts a <code>transition</code> prop you
-              can provide a{' '}
-              <a href="https://github.com/reactjs/react-transition-group">
-                react-transition-group@2.0.0
-              </a>{' '}
-              compatible
-              <code>Transition</code> component and it will work.
-            </p>
-
-            <p>
-              Feel free to use <code>CSSTransition</code> specifically, or roll
-              your own like the below example.
-            </p>
-
-            <Playground codeText={TransitionSource} />
-          </section>
-        </main>
-      </div>
-    );
-  }
+function App() {
+  return null;
```

Comment:
?

Reply:
what's the point of this?

Reply:
it still gets rendered when running locally sometimes, be we redirect immediately